sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/processor/LabelsProcessor.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.processor;
+
+import io.opentelemetry.api.metrics.common.Labels;
+import io.opentelemetry.context.Context;
+
+public interface LabelsProcessor {
+  /**
+   * Called when bind() method is called. Allows to manipulate labels which this instrument is bound
```

Comment:
Actually `bind()` is not very accurate, it is an implementation detail in our SDK that we implement both the "bounds" and "direct records" using the same internal `bind()` method in the accumulator. Probably you need to use API concepts when describing this instead of internal concepts in the SDK.

Reply:
tried to provide a more accurate description. I had to double check, but seems that bind() as well as synchronous instrument is a part of API concepts. I did get your point though and tried avoid using implementation details to describe the behavior